cmd/cluster-init/generate_job.go

```diff
@@ -0,0 +1,182 @@
+package main
+
+import (
+	"fmt"
+
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	prowapi "k8s.io/test-infra/prow/apis/prowjobs/v1"
+	prowconfig "k8s.io/test-infra/prow/config"
+
+	"github.com/openshift/ci-tools/pkg/api"
+)
+
+const (
+	LatestImage   = "registry.ci.openshift.org/ci/applyconfig:" + api.LatestReleaseName
+	OpenshiftRole = "ci.openshift.io/role"
+	Infra         = "infra"
+	Tmp           = "tmp"
+)
+
+func generatePeriodic(clusterName string) prowconfig.Periodic {
+	args := generateArgs(clusterName)
+	args = append(args, "--confirm=true")
+	trueBool := true
```

Comment:
Look into package `utilpointer "k8s.io/utils/pointer"`

Reply:
ah...cool. I was using this proto package to do this, but it changed it to a direct dependency in vendors, and I didn't feel like that was worth it.